CHANGELOG.md

```diff
@@ -1961,7 +1961,7 @@
   * Include ControllerInstrumentation module with ActiveSupport.on_load
 
   The agent will now use the `on_load :action_controller` hook to include
-  the ControllerInstrumentation module in to both the `Base` and `API`
+  the ControllerInstrumentation module into both the `Base` and `API`
```

Comment:
`in to` and `into` are the thing I've started to play w/ most recently. So far it has been by far the highest friction level of all items. If you have objections, I'll gladly drop it (and anything else, but definitely this).

Reply:
I think the single word makes more sense here. Seems like an interesting coding problem to solve!

Reply:
Yeah. I'm still letting the problem percolate. I expect it'll take me at least 6 months before I have any insight. For the time being, I just try to be humble and explain each time that I'm not sure and only offer the change where my personal taste seems to like it (often asking some peers for opinions before offering).